src/utils.jl

```diff
@@ -23,13 +23,6 @@ as they are treated separately.
 """
 asqtype(x::T) where {T<:Unitful.Units} = typeof(1.0*x)
 
-
-# Handle working with  Missings
-const UnitfulMissing = Union{<:Unitful.Quantity, Missings.Missing}
-
-*(y::Missings.Missing, x::T) where {T<:Unitful.FreeUnits} = y
```

Comment:
Why were these removed?

Reply:
The exist in Unitful now

Although I just checked and not until 0.15, so I need to change the Project.toml